viz-lib/src/visualizations/chart/plotly/fixtures/prepareData/heatmap/with-labels.json

```diff
@@ -36,7 +39,7 @@
         "showlegend": false,
         "text": ["3", "1", "4", "2"],
         "textfont": {
-          "color": ["black", "black", "black", "black"]
+          "color": ["#333333", "#ffffff", "#333333", "#ffffff"]
```

Comment:
@guidopetri 
This is the test case for the new annotation. `textfont.color` changes (`#333333` and `#FFFFFF`) according to background color now.

Reply:
Ohhh! I see now. Thanks for pointing this out!